extensions/browser-bookmarks/src/index.tsx

```diff
@@ -63,12 +63,13 @@ export default function Command() {
         return [browsers[0].bundleId as string];
       }
 
+      // TODO I don't understand why pulling the default is useful here
```

Comment:
We pull the default browser to enable it until the user manually selects the browsers from which they want to pull the bookmarks. If we remove that, then the user would see an empty state and would have to select browsers the very first time. To be honest, that would be okay from a UX standpoint, but having it already pre-selected is better since the user doesn't have to think about it.